server/server.go

```diff
@@ -23,32 +25,49 @@ const (
 	Version = "0.0.1"
 )
 
-type Mode handler.Mode
+type Mode int
 
 const (
-	DEV Mode = iota
-	PROD
+	ModeProd Mode = iota
+	ModeDev
 )
 
+func (m Mode) IsProd() bool {
+	switch m {
+	case ModeProd:
+		return true
+	default:
+		return false
+	}
+}
```

Comment:
Should we put this in `mode.go` file?

Reply:
Maybe. It is in fact used in different files.

Reply:
done